librz/bin/p/bin_mach0.c

```diff
@@ -35,7 +35,14 @@ static Sdb *get_sdb(RzBinFile *bf) {
 static char *entitlements(RzBinFile *bf, bool json) {
 	rz_return_val_if_fail(bf && bf->o && bf->o->bin_obj, NULL);
 	struct MACH0_(obj_t) *bin = bf->o->bin_obj;
-	return rz_str_dup(NULL, (const char *)bin->signature);
+	if (!bin->signature) {
+		return NULL;
+	}
+	if (json) {
+		return rz_str_newf("\"%s\"", (const char *)bin->signature);
```

Comment:
Use `rz_str_escape_utf8_for_json()`, or maybe even `PJ` with just a single `pj_s()` if that works.

Reply:
good suggestion, i'll try